packages/devtools-reps/src/reps/error.js

```diff
@@ -147,6 +156,12 @@ function getStacktraceElements(props, preview) {
       }, location));
     });
 
+  if (isStacktraceALongString) {
+    // Remove the last frame which is certainly incomplete.
+    // Can be removed when https://bugzilla.mozilla.org/show_bug.cgi?id=1448833 is fixed.
+    stack.splice(-2);
```

Comment:
The comment says "Remove the last frame", but using -2 means last two frames are removed, is that correct?

Reply:
yes, it's unclear here, but stack has 2 items for each frame (function name + location). I'll change that comment to make it more obvious.